libraries/botbuilder-adapter-twitter/src/schema.ts

```diff
@@ -0,0 +1,24 @@
+import * as Twitter from 'twitter';
+
+/**
+ * @module botbuildercommunity/adapter-twitter
+ */
+
+/*
+ * Settings below are in snake case because expected Twitter parameters.
+ */
+export interface TwitterAdapterSettings extends Twitter.AccessTokenOptions {
```

Comment:
Is this necessary? Could we not just export `Twitter.Options` as `TwitterAdapterSettings`? In that case we can add fields if necessary later, but currently it is not needed.

`Twitter.AccessTokenOptions` is extending `Twitter.Options`, which already has the `consumer_key` and `consumer_secret`.

Reply:
Didn't see the `Twitter.Options` version.

Reply:
Changed.

Reply:
I did also a bit more research, but I was not able to export `Twitter.AccessTokenOptions` as `TwitterAdapterSettings`. I guess because it is in the DefinitelyTyped package. I think it is nice to have it as an interface from the adapter itself, since you can add more adapter specific settings in the feature without introducing breaking changes? What do you think.

Reply:
I'll have a better handle on needs once I get a working example together. We can revisit it then.